demo/calibrationManager.js

```diff
@@ -0,0 +1,96 @@
+export async function storeCalibration(s3, bucket, name, callback) {
+  // TODO
+}
+
+export async function loadRtk2Vehicle(s3, bucket, name, callback) {
+
+  // TODO Hardcoded defaults for now
+  let rtk2Vehicle = {
+    x:0, y:0, z:-2,
+    roll:0, pitch:0, yaw:Math.PI/2
+  }
+
+  callback(rtk2Vehicle);
+}
+
+export async function loadVelo2Rtk(s3, bucket, name, callback) {
+  const tstart = performance.now();
+  //is name here the dataset name? We should be more careful about that....
+  if (s3 && bucket && name) {
+    (async () => {
+      const objectName = `${name}/7_Cals/extrinsics.txt`
+
+      try {
+        const data = await s3.getObject({Bucket: bucket, Key: objectName}).promise();
+        const calibrationText = new TextDecoder("utf-8").decode(data.Body);
+        const extrinsics = parseCalibrationFile(calibrationText);
+        callback( extrinsics );
+      } catch (err) {
+        console.log(err, err.stack)
+      }
+
+    })();
+
+  } else {
+    const filename = `../cals/extrinsics.txt`;
+    let t0, t1;
+
+    const xhr = new XMLHttpRequest();
+    xhr.open("GET", filename);
+    xhr.responseType = "text";
+
+    xhr.onprogress = function(event) {
+      t1 = performance.now();
+      t0 = t1;
+    }
+
+    xhr.onload = async function(data) {
```

Comment:
I don't think this needs to be `async`. It doesn't use `await`.

Reply:
Removed async declaration